### DISCUSSION TITLE
既存モンスターの調整案 (～v2.2.1)

### DISCUSSION THREAD

**Discussion by Hourier** [hengband/hengband#1169] [Ideas]
過去に実装済だったモンスターの調整案はこちらでお受けします
例：戦士系モンスターの打撃を「殴る」から「斬る」に変えて欲しい

(v3.0.0α以降、いくつかのモンスターに番号異動がありましたが、それは不問とします)

**Comment by dis-** [comment]
中盤の急所になりやすい<コーン>の血戮悪魔を虐殺対象から少し骨のあるステータスに改造したい。

N:523:<コーン>の血戮悪魔
E:Bloodletter of Khorne
G:U:r
I:127:40d8:20:60:30
W:34:1:0:600:0:0
B:SLASH:EXP_20:6d5
B:SLASH:EXP_20:6d5
B:SLASH:EXP_20:6d5
F:FORCE_MAXHP | OPEN_DOOR | FRIENDS | DROP_90 | REGENERATE |
F:ONLY_ITEM | NO_FEAR | NONLIVING |
F:EVIL | IM_POIS | IM_COLD | IM_FIRE | RES_CHAO | DEMON
D:$It is a slender, red-skinned demon twisted into a nightmarish shape.
D:$ It is armed with a hellblade, which will suck the life from your body.
D:流血の神<コーン>の配下の下級悪魔。
D:悪夢のような姿に捻じ曲がった、細身で赤い肌をした悪魔だ。
D:ヘルブレードを装備していて、その刃はあなたの生命力を
D:吸いとる。

**Reply by Hourier** [reply]
Discordの議論ざっくり見ました
パラメータ調整は問題ないです
デッドアタック枠でヘルブレードの召喚処理 (例：1/3 の確率で1体)も一緒にやろうと思えばできますが、どうしましょうか？

**Reply by dis-** [reply]
剣を2本落とすと違和感ありますが、カオスブレードを落とさなかった場合に一定確率でヘルブレード召喚をやるのは良いかもしれませんね。

**Reply by Hourier** [reply]
OKです
後ほど (α26目処)で調整してみます

**Comment by Hourier** [comment]
Discussionから正式にチケット化した
https://github.com/hengband/hengband/issues/1455

**Comment by ironside777** [comment]
闘技場で現状『らいおんはーと』にのみ賞品がありませんが、
特段の事情が無ければ、何か賞品を設定してはいかがでしょうか。
（discordで トラップ感知のロッド などの案はありました）

**Reply by Hourier** [reply]
アリですね
というか序盤数体のアイテムはワインだの装飾だの売る手間すらもったいないレベルのショボい景品なので、そこら辺まとめて調整してもいいかもしれません
まだ手が回らないですがいずれ検討しようと思います

**Comment by ironside777** [comment]
銀のエンゼルは5体でおもちゃのカンヅメが手に入るはずですが、
金のエンゼルよりは出やすいと言っても、レア過ぎて1ゲームに5体も見かけることがほぼありません。
出現率を上げてはいかがでしょうか。

**Reply by Hourier** [reply]
計画にはありつつもほったらかしていた調整です
後日対応します

**Comment by gold-onion** [comment]
### モンスターの弱点について
既存モンスターだけの話ではないんですが、
1. 思い出の表記では「～でダメージを与えられる。」ですが、処理が共通のため特定のモンスター以外でもダメージが入る炎や冷気でも同じ表記なので、ちょっと不自然な気がします。炎や冷気は「～で大きなダメージを与えられる。」とかに変更できないでしょうか。
2. 岩を除去するものでダメージを与えられるモンスターについて、採掘によるスレイを追加できないでしょうか。

**Comment by gold-onion** [comment]
インターネット・エクスプローダーにサポート終了のお知らせが欲しいです

**Comment by gold-onion** [comment]
ヨルムンガンドは遠距離攻撃が低頻度のウォーター・ボールしか無く、水耐性の導入で弱体化してるので、何かテコ入れできないでしょうか
